src/lib/airtable/tables/requests.js

```diff
@@ -83,10 +83,7 @@ exports.findDeliveryNeededRequests = async () => {
       })
       .all();
 
-    return [
-      requests.filter(r => !requestNotInSlack(r)).filter(notForDrivingCluster),
-      null
-    ];
+    return [requests.filter(r => !requestNotInSlack(r)), null];
```

Comment:
don't filter out driving cluster. TODO: also include driving cluster but *not* in slack

Reply:
this filtering was also to ensure create delivery request still works when >8 requests are assigned. I can follow up with a different issue tho!

Reply:
can you say more about `ensure create delivery request still works when >8 requests are assigned.`? in what way is it related to filtering out `notForDrivingCluster`?

Reply:
ah my mistake. this method is for requests that are already posted.